recipe/meta.yaml

```diff
@@ -24,6 +26,8 @@ requirements:
     - make  # [not win]
     - ninja  # [win]
     - pkg-config  # [linux and aarch64]
+    - patch  # [not win]
```

Comment:
as there are no patches anymore, we don't need build dependencies m2-patch, nor patch anymore